app/src/ui/delete-branch/delete-pull-request-dialog.tsx

```diff
@@ -29,13 +29,14 @@ export class DeletePullRequest extends React.Component<IDeleteBranchProps, {}> {
         onDismissed={this.props.onDismissed}
       >
         <DialogContent>
-          <p>This branch has an open pull request associated with it.</p>
+          <p>This branch may have an open pull request associated with it.</p>
           <p>
             If{' '}
             <LinkButton onClick={this.openPullRequest}>
               #{this.props.pullRequest.pullRequestNumber}
             </LinkButton>{' '}
-            has been merged, you can also remove the remote branch on GitHub.
+            has been merged, you can also go to GitHub.com to delete the remote
```

Comment:
> GitHub.com

It's possible that the PR is stored on a GitHub Enterprise Server instance and in that case I think explicitly saying `GitHub.com` could add some confusion. We can either use a non-specific host terminology or we can change the text if the PR is on GHES, I'll leave that up to you.

Reply:
```suggestion
            has been merged, you can also go to GitHub to delete the remote
```